OptimizelySDK/Optimizely.cs

```diff
@@ -645,5 +669,10 @@ private bool ValidateStringInputs(Dictionary<string, string> inputs)
 
             return isValid;
         }
+
+        private bool IsValidDecision(FeatureDecision decision)
```

Comment:
What is this validity check about? Also, I feel it belongs on the `FeatureDecision` class and not like this.

Reply:
Perhaps call it `IsExperimentDecision`?

Reply:
This check validates decision object. But I misunderstood that DecisionService.GetVariationForFeature() can return null.

I removed this function as the above method can't return null and checking decision.Variation is enough for validating decision object.